src/Hotel.java

```diff
@@ -5,24 +5,29 @@
     private String name;
     private ArrayList<Room> rooms;
 
-    Hotel(String name, ArrayList<Room> rooms) {
+    Hotel(String name, ArrayList<Room> rooms)
+    {
         this.name = name;
         this.rooms = rooms;
     }
 
-    String getName(){
+    String getName()
+    {
         return this.name;
     }
 
-    ArrayList<Room> getRooms(){
+    ArrayList<Room> getRooms()
+    {
         return this.rooms;
```

Comment:
You don't need to add this. to every class level property. 
When a method has a local property which has the same name as a class property, the method will always use the local property, because it is closer. That's why we use this. to explicitly say that we need the class level property.
In most methods of this class you don't need the this.

Reply:
added